### DISCUSSION TITLE
Make trails visible at wider zoom levels

### DISCUSSION THREAD

**Discussion by jacob-i-skinner** [organicmaps/organicmaps#5541] [Ideas]
I recently used organic maps for route planning and navigation during some wilderness hiking. It's offline capabilities make it very good for such application, but the map readability are not so good especially at wide zoom levels. making trails and unpaved road visible at wider zoom levels could make the app much more usable for route planning.

EDIT: removed references to line thickness for clarity.

**Comment by biodranik** [comment]
Is it about making thinner lines thicker, or about displaying trails where they are not displayed now? Any screenshots/examples would be helpful. CC @pastk

**Comment by jacob-i-skinner** [comment]
Mostly I am concerned with displaying trails where they are currently not displayed. My preference would be for the trail and unpaved road to trailhead (both visible in the last screenshot) to both be visible from further out, e.g. the level of the first screenshot. 

![Screenshot_20230715-111927](https://github.com/organicmaps/organicmaps/assets/18603331/56831806-1cfd-49a8-9274-548d4ce7d4ee)
![Screenshot_20230715-112052](https://github.com/organicmaps/organicmaps/assets/18603331/2a1c186c-b80e-4dcd-bae3-5ba10839c655)
![Screenshot_20230715-112111](https://github.com/organicmaps/organicmaps/assets/18603331/15ee4e99-5de1-4748-aa6d-d2ea5c36e410)

**Comment by pastk** [comment]
> Mostly I am concerned with displaying trails where they are currently not displayed. My preference would be for the trail and unpaved road to trailhead (both visible in the last screenshot) to both be visible from further out, e.g. the level of the first screenshot.
> 

Its planned in the outdoors map style https://github.com/organicmaps/organicmaps/issues/2356.
There was a beta preview of it in April #4247.

**Reply by jacob-i-skinner** [reply]
Thank you. Closing this idea as duplicate.
